week1/project/ecommerce/src/categories.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import allCategories from "./fake-data/all-categories.js";
+
+const Categories = ({ onSelectCategory }) => {
+  const categories = allCategories;
+
+  const handleSelectCategory = (categoryId) => {
+    onSelectCategory(categoryId);
+  };
+
+  return (
+    <div className="categories">
+      {categories.map((category) => (
+        <button
+          key={category.id}
+          onClick={() => handleSelectCategory(category.id)}
+        >
+          {category.name}
```

Comment:
@OmerAlimoglu Can you use `category` only 
```suggestion
          {category}
```